rclcpp/include/rclcpp/service.hpp

```diff
@@ -486,6 +486,14 @@ class Service
   {
     rcl_ret_t ret = rcl_send_response(get_service_handle().get(), &req_id, &response);
 
+    if (ret == RCL_RET_TIMEOUT) {
+      RCLCPP_WARN(
+        rclcpp::get_node_logger(node_handle_.get()).get_child("rclcpp"),
```

Comment:
why not using `node_logger_` here?

Reply:
I'm not sure that there is access to the node logger in this class right now.

Reply:
@kghost can you address this one? and then i think we are good to go.